test/Directory.Packages.props

```diff
@@ -0,0 +1,138 @@
+    <!-- <PackageVersion Include="Grpc.Tools" Version="2.44.0" /> -->
+    <!-- Workaround! Microsoft.Extensions.Logging.Abstractions v.3.1.0 is minimal version supported by auto instrumentation.
+    Grpc.Net.Client references older version. It prevents to load required version from Additional Dependencies store-->
+    <PackageVersion Include="Microsoft.Extensions.Logging.Abstractions" Version="3.1.0" />
+  </ItemGroup>
+  
+  <!-- Versions from TestApplication.MassTransit.csproj -->
+  <ItemGroup>
+    <PackageVersion Include="Microsoft.Extensions.Hosting" Version="6.0.0" />
+    <PackageVersion Include="MassTransit" Version="8.0.0" />
+  </ItemGroup>
+
+  <!-- Duplicate -->
+  <!-- Versions from TestApplication.Modules.csproj
+  <ItemGroup>
+    <PackageVersion Include="Newtonsoft.Json" Version="13.0.2" />
+  </ItemGroup> -->
+
+  <!-- TODO: complicated logic here. Is it better to not centralize the package version for MongoDB.csproj? -->
+  <!-- Versions from TestApplication.MongoDB.csproj -->
+  <PropertyGroup>
+    <MongoDBDriverApiVersion Condition="'$(MongoDBDriverApiVersion)' == ''">2.13.3</MongoDBDriverApiVersion>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageVersion Include="MongoDB.Driver" Version="$(MongoDBDriverApiVersion)" />
+    <!-- Workaround! Microsoft.Extensions.Logging.Abstractions v.3.1.0 is minimal version supported by auto instrumentation.
+    MongoDB.Driver 2.18+ references older version. It prevents to load required version from Additional Dependencies store-->
+    <!-- <PackageVersion Include="Microsoft.Extensions.Logging.Abstractions" Version="3.1.0" Condition="'$(MongoDBDriverApiVersion)'>='2.18.0'" /> -->
+  </ItemGroup>
+
+  <!-- Versions from TestApplication.MySqlData.csproj -->
+  <ItemGroup>
+    <PackageVersion Include="MySql.Data" Version="6.10.7" />
+  </ItemGroup>
+
+  <!-- Versions from TestApplication.Npgsql.csproj -->
+  <ItemGroup>
+    <PackageVersion Include="Npgsql" Version="6.0.5" />
+  </ItemGroup>
+
+  <!-- Versions from TestApplication.NServiceBus.csproj -->
+  <ItemGroup>
+    <PackageVersion Include="Microsoft.Extensions.DependencyInjection.Abstractions" Version="7.0.0" />
+    <PackageVersion Include="NServiceBus" Version="8.0.0" />
+  </ItemGroup>
+
+  <!-- Versions from TestApplication.Plugins.csproj -->
+  <ItemGroup>
+    <PackageVersion Include="OpenTelemetry" Version="1.4.0-rc.2" />
+    <PackageVersion Include="OpenTelemetry.Exporter.OpenTelemetryProtocol" Version="1.4.0-rc.2" />
+    <PackageVersion Include="OpenTelemetry.Instrumentation.Http" Version="1.0.0-rc9.11" />
+  </ItemGroup>
+
+  <!-- Versions from TestApplication.Smoke.csproj -->
+  <ItemGroup>
+    <PackageVersion Include="Microsoft.Extensions.Logging" Version="7.0.0" />
+  </ItemGroup>
+
+  <!-- TODO: VersionOverride: reason unknown for why this csproj used a lower version. Need to verify if latest version works. -->
```

Comment:
Same here, outside the scope of the PR. Use the version from test app.

Reply:
Done.